sqlmesh/core/model/meta.py

```diff
@@ -78,6 +78,7 @@ class ModelMeta(_Node):
     signals: t.List[exp.Tuple] = []
     enabled: bool = True
     physical_version: t.Optional[str] = None
+    gateway: t.Optional[str] = None
```

Comment:
Should this be a part of the data hash?

Reply:
Yes I think so I already have it in the data hash